caiman/base/movies.py

```diff
@@ -1145,7 +1146,9 @@ def animate(i):
 
 
 
-def load(file_name,fr=30,start_time=0,meta_data=None,subindices=None,shape=None, var_name_hdf5 = 'mov', in_memory = False, is_behavior = False, bottom=0, top=0, left=0, right=0, channel = None):
+def load(file_name,fr=30,start_time=0,meta_data=None,subindices=None,shape=None, 
+         var_name_hdf5 = 'mov', in_memory = False, is_behavior = False, bottom=0, 
+         top=0, left=0, right=0, channel = None, outtype=np.float32):
```

Comment:
Do you have an intended use in mind for this new parameter? Right now everything is still using np.float32